ctrd/snapshot.go

```diff
@@ -31,8 +31,13 @@ func (c *Client) CreateSnapshot(ctx context.Context, id, ref string) error {
 		return err
 	}
 
+	var opts []snapshots.Opt
+	if labels != nil && len(*labels) > 0 {
+		opts = append(opts, snapshots.WithLabels(*labels))
```

Comment:
How about just `opts := []snapshots.Opt{snapshots.WithLabels(labels)}`, of course, not use `*map` here, see last comment

Reply:
done